tests/unit/test_sql_execution.py

```
@@ -263,14 +283,25 @@ def test_execute_sql_with_connection_json_with_snowflake_private_key(
     def test_execute_sql_with_connection_json_with_snowflake_encrypted_private_key(
         self, mock_execute_sql_with_caching
     ):
+        private_key = rsa.generate_private_key(public_exponent=65537, key_size=2048)
+        private_key_passphrase = base64.b64encode(random.randbytes(16)).decode("utf-8")
```

Ruff (0.14.8)

Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)